### DISCUSSION TITLE
filesystem MCP configuration failed

### DISCUSSION THREAD

**Discussion by ceasarXuu** [modelcontextprotocol/.github#25] [General]
SQLite and filesystem were configured according to the instructions;
sqlite can be used successfully, but filesystem does not seem to be installed successfully.
There is no problem with the file path. I tried starting claude with administrator rights but it also failed.

![image](https://github.com/user-attachments/assets/62ec3687-75e5-468c-baa5-d26cd30f0510)

![image](https://github.com/user-attachments/assets/f2a2f2d5-7f06-4912-8b1b-90d8cf2cc6fb)

![image](https://github.com/user-attachments/assets/5ac55acb-3c9a-4dee-886b-f2c4f4bd7e8f)

![image](https://github.com/user-attachments/assets/fb74d7f5-d0bf-4c49-9273-0362ad2f992e)

**Comment by philgei** [comment]
I got the same problem. It seems like all the Typescript based Servers cant be run. Both SqLite and Fetch are Python based (and therefore use UVX) and work.

**Comment by theUpsider** [comment]
@philgei and me wrote a small unofficial port to python: https://github.com/philgei/mcp_server_filesystem
let me know if this works for you.

**Comment by jspahrsummers** [comment]
Sorry about the trouble! I think this is the same as https://github.com/modelcontextprotocol/servers/issues/40, which we're working on a fix for. Will merge the discussion there.
